cmd/cli/app/verify.go

```diff
@@ -126,16 +150,17 @@ var verifyCmd = &cobra.Command{
 				right = result
 			}
 			result = hasher.HashChildren(left, right)
-			fmt.Printf("SHA256(0x01 | %v | %v) =\n\t%v\n\n", hex.EncodeToString(left), hex.EncodeToString(right), hex.EncodeToString(result))
+			o.Hashes = append(o.Hashes,
```

Comment:
do you really want to have this be the output within the JSON structure too, or just have the raw hashes? I might vote for keeping the hashes as what is in the return object and moving the bulk of this code into String().

Reply:
Good call. Fixed!